### PR TITLE
helm chart: Allow users to set extra env by external configmap

### PR DESCRIPTION
**Pull request by lianhao**
## Description

- Allow users to set extra env by external configmap through global.extraEnvConfig
- export data-prep configuration items into its value file
- bump chart version to upcoming 0.8.0 and adapted to v0.7 opea docker images

## Issues

`n/a`

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)

## Dependencies

n/a

## Tests

verified on Xeon with recently published opea docker image(v0.7)

### CODE REVIEW

File: helm-charts/common/llm-uservice/values.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ podSecurityContext: {}
   # fsGroup: 2000
 
 securityContext:
-  readOnlyRootFilesystem: true
+  readOnlyRootFilesystem: false
```
Comment (yongfengdu) [inline helm-charts/common/llm-uservice/values.yaml:27]: Any reason the flag is set back to false?

Comment (lianhao) [reply helm-charts/common/llm-uservice/values.yaml:27]: because latest llm-uservice need to install python module upon start time

**Review by daisy-ycguo — APPROVED** [review]
lgtm

File: helm-charts/common/llm-uservice/values.yaml (hunk 1 above)
Comment (yongfengdu) [reply helm-charts/common/llm-uservice/values.yaml:27]: OK, https://github.com/opea-project/GenAIComps/pull/274 made that change.

**Review by yongfengdu — APPROVED** [review]

**Pull request merged**